spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -34,8 +84,70 @@ When running in an environment where Jakarta Transactions and Jakarta CDI are av
 
 When a repository method is annotated with an interceptor binding annotation, the interceptor is bound to the repository bean according to the interceptor binding annotation of the repository interface method, causing the bound interceptor to be invoked around the repository method when it runs. This enables the use of interceptors such as `jakarta.transaction.Transactional` on repository methods when running in an environment where the Jakarta EE technology that provides the interceptor is available.
 
-=== Jakarta Persistence
+=== Mapping an Entity
+
+An entity class in Jakarta Data is a regular Java class that is annotated to indicate its ability to represent objects in a database. Jakarta Data itself does not provide specific annotations for mapping entities. Instead, each Jakarta Data provider can create annotations or support existing ones, such as those used in Jakarta Persistence or Jakarta NoSQL.
+
+==== Jakarta Persistence
+
+Jakarta Persistence is a Jakarta EE specification that provides an object-relational mapping (ORM) framework for Java applications. It aims to simplify the development of Java applications that interact with relational databases.
```

Comment:
I think we should remove these spec descriptions at the start of each section. They're nice descriptions, but I'd prefer to keep the spec concise, and I would expect spec users to either already be familiar with the other specifications, or look at the other specs for a description.

I looked at how the Bean Validation and Persistence specs talk about integrations, and they jump right into the integration without a description.